website/docs/admin_hosts_houdini.md

```diff
@@ -3,9 +3,26 @@ id: admin_hosts_houdini
 title: Houdini
 sidebar_label: Houdini
 ---
+## General Settings
+### JOB Path
+you can add your studios preffered JOB Path, JOB value will be checked and updated on file save and open.
```

Comment:
I'd probably phrase more ont he lines of:
```md
Specify a studio-wide `JOB` path ...
``` 

There's a typos:
 `Disableing` -> `Disabling`; 
`effectivly` -> `effectively`